src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work.